libs/libtalloc/Makefile

```diff
@@ -6,19 +6,21 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=talloc
-PKG_VERSION:=2.1.14
+PKG_VERSION:=2.3.1
 MAJOR_VERSION:=2
-PKG_RELEASE:=3
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://www.samba.org/ftp/talloc
-PKG_HASH:=b185602756a628bac507fa8af8b9df92ace69d27c0add5dab93190ad7c3367ce
+PKG_HASH:=ef4822d2fdafd2be8e0cabc3ec3c806ae29b8268e932c5e9a4cd5585f37f9f77
 
 PKG_MAINTAINER:=Ted Hess <thess@kitschensync.net>
 PKG_LICENSE:=LGPL-3.0-or-later
 
 PKG_BUILD_PARALLEL:=0
 
+HOST_BUILD_DEPENDS:=python/host
```

Comment:
This is Python 2 :joy:

Are you sure this is necessary? The system python3 is symlinked into `$(STAGING_DIR_HOST)/host/bin` (both `python` and `python3`), and if a "fuller" host Python is needed then you probably want `HOST_BUILD_DEPENDS:=python3/host`.

Reply:
This line is probably no-op (host-build.mk is not included). But yes this was a mistake that crept in.

Reply:
You're probably right - my reaction to seeing `python/host` blinded me to everything else :laughing:

Reply:
the line was added (328f60266e81021) when isolated builds for libtalloc where failing before the update.
i'm also very unsure about the changes in cross-answers.